subprojects/gradle/design-screenshots/src/main/kotlin/com/avito/plugin/DeviceProvider.kt

```diff
@@ -30,7 +30,7 @@ internal class DeviceProviderLocal(
         return adbDeviceFactory.create(
             coordinate = DeviceCoordinate.Local(serial),
             adbDeviceParams
-        )
+        ).getOrThrow()
```

Comment:
For local still fail because I don't know how hadle that

Reply:
I know it's cool to support local running, but we probably should drop it altogether for now: no use cases = accumulated tech debt and expensive support